doc/user_doc/vic_installation/vch_installer_examples.md

```diff
@@ -0,0 +1,193 @@
+# Examples of Deploying a Virtual Container Host #
+
+This topic provides examples of the `vic-machine` options to use when deploying virtual container hosts in different vSphere configurations.
+
+- [Deploy a Virtual Container Host Directly on an ESXi Host with no Resource Pools](#esxi)
+- [Deploy a Virtual Container Host in a vCenter Server Cluster](#cluster)
+- [Deploy a Virtual Container Host to a Cluster and Specify External, Management, and Container Networks](#networks)
+- [Deploy a Virtual Container Host and Specify a Different Datastore for Container VM Files](#datastores)
+- [Deploy a Virtual Container Host on a Standalone Host in vCenter Server](#standalone)
+- [Deploy a Virtual Container Host in a Resource Pool on an ESXi Host](#rp_host)
+- [Deploy a Virtual Container Host in a Resource Pool in a vCenter Server Cluster](#rp_cluster)
+- [Deploy a Virtual Container Host for Use with a Docker Client that Requires CA Certificates](#ca_cert)
+- [Deploy a Virtual Container Host with no TLS Authentication](#no_tls)
+- [Deploy a Virtual Container Host with Customized VM Settings](#customized)
+
+<a name="esxi"></a>
+## Deploy a Virtual Container Host Directly on an ESXi Host with no Resource Pools##
+
```

Comment:
@mdubya66, you mean link to the ESXi and vCenter Server docs? Since this doc is aimed at vSphere Admins, I'm pretty certain that they won't need to look at docs about clusters, resource pools, etc. Personally, I think that would be overkill.